XTS/WSTX/classes/com/arjuna/mwlabs/wst11/at/remote/UserTransactionImple.java

```diff
@@ -288,7 +288,7 @@ protected final Context startTransaction(int timeout, TxContextImple current)
             //InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream("/foo.properties");
 
 
-            final Long expires = (timeout > 0 ? new Long(timeout) : null) ;
+            final Long expires = (timeout > 0 ? Long.valueOf(timeout) : null) ;
```

Comment:
Unnecessary boxing, use `(long) timeout` instead. Similarly elsewhere in this PR.